pkg/client/pvmclient.go

```diff
@@ -85,15 +84,11 @@ func NewPVMClient(c *Client, instanceID, instanceName, ep string) (*PVMClient, e
 
 	pvmclient.InstanceName = svc.Name
 	pvmclient.Zone = svc.RegionID
-	pvmclient.Region, err = utils.GetRegion(pvmclient.Zone)
-	if err != nil {
-		return nil, err
-	}
 
 	authenticator := &core.IamAuthenticator{ApiKey: c.Config.BluemixAPIKey, URL: *c.Config.TokenProviderEndpoint}
 
 	if power_api_endpoint := os.Getenv("IBMCLOUD_POWER_API_ENDPOINT"); power_api_endpoint == "" {
-		os.Setenv("IBMCLOUD_POWER_API_ENDPOINT", fmt.Sprintf("%s.%s", pvmclient.Region, ep))
+		os.Setenv("IBMCLOUD_POWER_API_ENDPOINT", ep)
```

Comment:
this change will break the current flow, lets understand who is using this env or when this introduced and handle the new change

Reply:
The env variable is used to set the service URL
https://github.com/IBM-Cloud/power-go-client/blob/master/ibmpisession/session.go#L81

Even when the env var is set without region,Its handled in the code to add the region in power api endpoint
https://github.com/IBM-Cloud/power-go-client/blob/master/ibmpisession/session.go#L89

So these changes will not break the existing flow.